Add hackathon result (#60)
web3community/DEV-NFT: README.md

This project aims for [Buidlit](https://buidlit.polygon.technology/) & [Dev Dapp Starter Grant](https://www.notion.so/Welcome-to-DEV-DAPP-STARTER-GRANTS-5cb95252f18540258111581ea54d8808)

> # Hackaton results

Reviewer: ```suggestion
# Hackathon results
```

> This project win 2 prizes for [Buidlit](https://buidlit.polygon.technology/) hackaton 🏆 :

Reviewer: ```suggestion
This project won 2 prizes for [Buidlit](https://buidlit.polygon.technology/) hackathon 🏆 :
```

> <img width="1259" alt="Capture d’écran 2021-12-27 à 16 15 30" src="https://user-images.githubusercontent.com/50140834/147484853-81c5bae0-8a8a-40d8-bdcb-f50b795f59d1.png">

Reviewer: ```suggestion
<img width="1259" alt="" src="https://user-images.githubusercontent.com/50140834/147484853-81c5bae0-8a8a-40d8-bdcb-f50b795f59d1.png">
```

> Check the project submission [here](https://devfolio.co/submissions/devnft-210b) !

Reviewer: ```suggestion
Check the project submission [here](https://devfolio.co/submissions/devnft-210b)!
```
